src/parseTools.mjs

```diff
@@ -1086,6 +1094,7 @@ function ENVIRONMENT_IS_WORKER_THREAD() {
 addToCompileTimeContext({
   ATEXITS,
   ATINITS,
+  ATPOSTINITS,
```

Comment:
Can we just not export the array itself?

I'm not sure library code needs to access the ATPOSTINITS does it?